src/config/dynamic-theme-fixes.config

```diff
@@ -4688,6 +4688,38 @@ INVERT
 
 ================================
 
+dnd.su
```

Comment:
Is this the correct domain? I'm not able to visit it.

Reply:
![image](https://user-images.githubusercontent.com/11363215/162589008-d84d82a6-92e8-4b83-bbb2-cc0d4dcfe251.png)
Yeah. It pretty is.